content/contributing/project-information/working-groups.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Working Groups"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+Working groups are temporary community initiatives devoted to particularly, complex, large, or important tasks. Some examples include:
+
+- Revamping Bevy's color APIs
+- Replacing Bevy's render graph
+- Implementing relations
+- And drafting this Contributing Guide!
```

Comment:
This is too specific and is going to get outdated very quickly. In a few Bevy releases (hopefully, if the WGs are successful at their goals), hardly anyone will remember these exact initiatives.

IMO this list should be written to be more general.

Something like

 - Reworking a tricky or flawed part of Bevy
 - Designing a new API for a major area of the engine
 - Implementing a large feature that requires complex coordination and review
 - Initiatives outside of programming, like writing a book
 - Project management efforts, like release prep

Reply:
Yeah, good idea.